tests/modules/config/home-cursor/default.nix

```diff
@@ -1,8 +1,24 @@
 {
   # Ensure backwards compatibility with existing configs
-  home-cursor-legacy = { realPkgs, ... }: {
+  home-cursor-legacy = { ... }: {
```

Comment:
this was supposed to be the test that works with setting a config without enable and threw a warning that the behavior was deprecated